src/js/components/Twitter/TwitterSignIn.jsx

```diff
@@ -5,17 +5,24 @@ import { oAuthLog, renderLog } from '../../utils/logging';
 import $ajax from '../../utils/service';
```

Comment:
Please revert this entire file from your pull request, since I believe the only changes are from auto-formatting: 
src/js/components/Twitter/TwitterSignIn.jsx